dts/bindings/ethernet/microchip,enc28j60.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2018, Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: 10Base-T Ethernet Controller with SPI Interface
+version: 0.1
+
+description: >
+    This binding gives a base representation of the standalone ENC28J60 chip
+inherits:
+  !include ethernet.yaml
```

Comment:
Including this will create long names (eth+spi) such as:
DT_SNPS_DESIGNWARE_SPI_B0001400_MICROCHIP_ENC28J60_B0001400_ENC28J60_LOCAL_MAC_ADDRESS_5

I see no benefit, the information it adds can be found in spi generated device tree definitions.

Reply:
I understand, but with in general spi devices should inherit from spi-device.yaml to get the standard properties of a SPI device.

Additionally with the new generation we now get #defines that are just DT_<COMPAT>_<INSTANCE>_<PROP> that we should update this driver to utilize.